### DISCUSSION TITLE
More Activity types

### DISCUSSION THREAD

**Discussion by MariothonsYT** [maximmax42/Discord-CustomRP#221] [Ideas]
For some programs on Discord, it will show a different activity type instead of showing just "Playing xyz" (Eg: Streaming,
Competing, Watching, Listening)

My idea is that this also becomes a selectable option. This is already available in multiple other rp programs as well.
There are 5 options and it shows as follows:
Playing - Playing XYZ
Streaming - Streaming XYZ
Watching - Watching XYZ
Listening - Listening to XYZ
Competing - Competing in XYZ

**Comment by maximmax42** [comment]
Requires obtaining user's token. Not implementing.
Might as well use Vencord + CustomRPC plugin at this point.

**Reply by rikoyoung** [reply]
> Requires obtaining user's token. Not implementing. Might as well use Vencord + CustomRPC plugin at this point.

I wanted to ask if it's possible to add a custom activity. Because I would like to use something like "Playing on Samsung Galaxy"

Thank you

**Reply by maximmax42** [reply]
> I wanted to ask if it's possible to add a custom activity. Because I would like to use something like "Playing on Samsung Galaxy"
> 
> Thank you

No, the types are predefined. You can call your presence app "on Samsung Galaxy", so it shows "**Playing** on Samsung Galaxy", but that's it.

**Reply by rikoyoung** [reply]
Alright, got it
